src/main/java/org/broadinstitute/hellbender/utils/bigquery/StorageAPIAvroReader.java

```diff
@@ -22,11 +27,9 @@
 
     private static final Logger logger = LogManager.getLogger(StorageAPIAvroReader.class);
 
-    private static int rowCount = 0;
+    private BigQueryReadClient client;
```

Comment:
now that google renamed storageAPI to ReadAPI, should we rename this class? not super important.